trio/_socket.py

```diff
@@ -62,54 +46,13 @@ def _is_blocking_io_error(self, exc):
 # CONSTANTS
 ################################################################
 
-# Hopefully will show up in 3.7:
-#   https://github.com/python/cpython/pull/477
-if not hasattr(_stdlib_socket, "TCP_NOTSENT_LOWAT"):  # pragma: no branch
-    if _sys.platform == "darwin":
-        TCP_NOTSENT_LOWAT = 0x201
-        __all__.append("TCP_NOTSENT_LOWAT")
-    elif _sys.platform == "linux":
-        TCP_NOTSENT_LOWAT = 25
-        __all__.append("TCP_NOTSENT_LOWAT")
-
-for _name in _stdlib_socket.__dict__.keys():
-    if _name == _name.upper():
-        _reexport(_name)
-
-if _sys.platform == "win32":
-    # See https://github.com/python-trio/trio/issues/39
-    # (you can still get it from stdlib socket, of course, if you want it)
-    globals().pop("SO_REUSEADDR", None)
-    __all__.remove("SO_REUSEADDR")
-
+try:
+    from socket import IPPROTO_IPV6
+except ImportError:
     # As of at least 3.6, python on Windows is missing IPPROTO_IPV6
     # https://bugs.python.org/issue29515
-    if not hasattr(_stdlib_socket, "IPPROTO_IPV6"):  # pragma: no branch
+    if _sys.platform == "win32":  # pragma: no branch
```

Comment:
According to https://codecov.io/gh/python-trio/trio/pull/670/changes you don't need `pragma: no branch`: this is covered by the Windows tests.

Reply:
@pquentin I removed the comment. How do I learn about codecov? What would be some good resources to read?

Reply:
@jmfrank63 Nothing that I know of, sorry :(